review.py

```diff
@@ -39,7 +39,7 @@ def get_review():
     top_p=1.0,
     frequency_penalty=0.0,
     presence_penalty=0.0
-  )
+  ) #this si bad
```

Comment:
is?

Reply:
Done on purpose to see if it catches it :)

Reply:
It caught the commment issue [here](https://github.com/mono-chrome/GPTReviewWorkflow/pull/4#pullrequestreview-1247756228)